### DISCUSSION TITLE
Creating signatures

### DISCUSSION THREAD

**Discussion by euluizaraujo** [juspay/hyperswitch#2936] [Q&A] [answered]
Hello, can I currently use hype for subscription charges? I didn't see anything specifically talking about this, but from what I understand, the mandates are for this, right?

**Comment by VedantKhairnar** [comment, marked as answer]
Hey @euluizaraujo
For recurring payments, you can use Mandates.  You will find more info around this [here](https://hyperswitch.io/docs/paymentMethods/cards#recurring-payments---mandate-through-cards)
Hyperswitch doesn't yet support subscription use cases like plan creation, modification etc, in case that is your need.

**Comment by euluizaraujo** [comment]
Hi @VedantKhairnar thanks for the feedback. So I could integrate, for example, with https://github.com/getlago/lago to meet this subscription management need and use mandates for recurring charges, right?

**Reply by manojradhakrishnan** [reply]
Yes. You can do that. If the process is documented and published, it will be useful for the broader community using both open source projects. We will be grateful if you are willing to give it a shot. 

Reference to similar discussion: https://github.com/juspay/hyperswitch/discussions/2821

Please let us know if interested to submit a PR.
